src/components/Main/Main.css

```diff
@@ -0,0 +1,18 @@
+.country {
+  display: flex;
+  align-items: center;
+  align-content: flex-start;
+  flex-direction: column;
```

Comment:
this is kind of a weird orientation for this kind of content, generally with a list of things we'll read it left to right rather than vertically since that's how english writing works